src/pke/include/cryptocontext.h

```diff
@@ -2701,6 +2689,17 @@ class CryptoContextImpl : public Serializable {
     std::shared_ptr<std::map<usint, EvalKey<Element>>> EvalSumRowsKeyGen(const PrivateKey<Element> privateKey,
                                                                          const PublicKey<Element> publicKey = nullptr,
                                                                          usint rowSize = 0, usint subringDim = 0);
+
+    // [[deprecated(
```

Comment:
Why is the deprecation logic commented out here and other places?

Reply:
It is the new code waiting to be un-commented after publicKey is removed. I can remove it

Reply:
This is fine. It is OK to keep it as is.